src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useContext } from 'react';
+import classNames from 'classnames';
+import { TodosContext } from '../../TodosContext/TodosContext';
+import { Status } from '../../types/Status';
+
+export const Footer: React.FC = () => {
+  const {
+    todos, deleteCompletedTodos, filterBy, setFilterBy,
+  } = useContext(TodosContext);
+
+  const notCompletedTasksCount = todos.filter(todo => todo.completed === false);
+  const completedTasksCount = todos.filter(todo => todo.completed === true);
+
+  const handleDeleteCompleted = () => {
+    deleteCompletedTodos();
+  };
```

Comment:
instead of creating this additional function  `handleDeleteCompleted `

you can use ` deleteCompletedTodos` directly on  onClick